packages/ripple-nuxt-tide/modules/publication/tide.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
       'field_landing_page_contact.field_paragraph_phones',
       'field_landing_page_contact.field_paragraph_social_media',
       'field_license_type',
-      'field_landing_page_component',
```

Comment:
So this can be removed?

Reply:
nope, did that by mistake, I have added it back.